store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/FederatedOperationTest.java

```diff
@@ -16,69 +16,96 @@
 
 package uk.gov.gchq.gaffer.federatedstore.operation;
 
-import org.apache.commons.lang3.exception.CloneFailedException;
 import org.junit.jupiter.api.Test;
 
-import uk.gov.gchq.gaffer.operation.Operation;
-import uk.gov.gchq.gaffer.operation.OperationChain;
-import uk.gov.gchq.gaffer.operation.impl.DiscardOutput;
+import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.jsonserialisation.JSONSerialiser;
 import uk.gov.gchq.gaffer.operation.impl.get.GetAdjacentIds;
-import uk.gov.gchq.gaffer.operation.impl.get.GetElements;
 
-import java.util.Map;
-
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-public class FederatedOperationTest {
-
-    private static class TestFederatedOperation implements FederatedOperation {
-
-        @Override
-        public Operation shallowClone() throws CloneFailedException {
-            return null;
-        }
+import java.util.List;
+import java.util.Set;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singleton;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static uk.gov.gchq.gaffer.federatedstore.util.FederatedStoreUtil.getHardCodedDefaultMergeFunction;
+
+public class FederatedOperationTest extends FederationOperationTest<FederatedOperation> {
+    private static final List<String> EXPECTED_GRAPH_IDS = asList("testGraphID1", "testGraphID2");
+    public static final String JSON = "{\n" +
```

Comment:
Because of the \n this will fail on Windows. Can we either remove the \n, or mark this class as @DisabledOnOs(OS.WINDOWS) as we have done with other tests.